src/cli/msig/staking.ts

```diff
@@ -148,6 +155,62 @@ export async function stakingContractSetup(options: StandardProposalOptions) {
         },
     };
 
+    // const buyRamAction = {
+    //     account: 'tonomy',
+    //     name: 'buyram',
+    //     authorization: [
+    //         {
+    //             actor: contract,
+    //             permission: 'active',
+    //         },
+    //     ],
+    //     data: {
+    //         dao_owner: 'ops.tmy',
+    //         app: contract,
+    //         quant: tokens,
+    //     },
+    // };
+
+    const setres = {
+        authorization: [
+            {
+                actor: 'tonomy',
+                permission: 'active',
+            },
+        ],
+        account: 'tonomy',
+        name: 'setresparams',
+        data: {
+            ram_price: RAM_PRICE,
+            total_ram_available: TOTAL_RAM_AVAILABLE,
+            ram_fee: RAM_FEE,
+        },
+    };
+
+    const actions = [adminSetAppAction, setres];
+
+    const proposalHash = await createProposal(
+        options.proposer,
+        options.proposalName,
+        actions,
+        options.privateKey,
+        [...options.requested, contract],
+        options.dryRun
+    );
+
+    if (options.dryRun) return;
+    if (options.autoExecute) await executeProposal(options.proposer, options.proposalName, proposalHash);
```

Comment:
why is this code change in this PR? seems nothing to do with the app colour changes